pkg/apis/core/types_shoot.go

```diff
@@ -613,6 +613,8 @@ type Alerting struct {
 // Provider contains provider-specific information that are handed-over to the provider-specific
 // extension controller.
 type Provider struct {
+	// EnableContainerD indicate if kubelet should be started with ContainerD and not docker
+	EnableContainerD bool
```

Comment:
Same like for v1beta1, v1alpha1 version

Reply:
Once we will decide i will also update here.